App.tsx

```diff
@@ -52,20 +52,43 @@ import {toISOStringUTC} from 'utils/date';
 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
 if (Constants.expoConfig.extra!.log_requests) {
   axios.interceptors.request.use(request => {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    if (!request.url.includes(Constants.expoConfig.extra!.log_requests_matching)) {
+      return request;
+    }
     console.log(
       'Request:',
       JSON.stringify(
         {
           method: request.method,
           url: request.url,
           params: request.params,
+          data: request.data,
         },
         null,
         2,
       ),
     );
     return request;
   });
+  axios.interceptors.response.use(response => {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    if (!response.config.url.includes(Constants.expoConfig.extra!.log_requests_matching)) {
+      return response;
+    }
+    console.log(
+      'Response:',
+      JSON.stringify(
+        {
+          status: response.status,
+          data: response.data,
+        },
+        null,
+        2,
+      ),
+    );
+    return response;
+  });
```

Comment:
is this super noisy?

Reply:
Yes, but when something's goofy, you need it - should we have a separate toggle for inbound & outbound requests?

Reply:
yeah, feels like `log requests/log responses` should be two different settings